src/Expressions/ExpressionBuilderInterface.php

```diff
@@ -1,20 +1,10 @@
 <?php
-/**
- * @link http://www.yiiframework.com/
- *
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
 
-namespace Yiisoft\Db;
+namespace Yiisoft\Db\Expressions;
 
 /**
  * Interface ExpressionBuilderInterface is designed to build raw SQL from specific expression
- * objects that implement [[ExpressionInterface]].
- *
- * @author Dmitry Naumenko <d.naumenko.a@gmail.com>
- *
- * @since 2.0.14
+ * objects that implement {@see ExpressionInterface}.
  */
 interface ExpressionBuilderInterface
```

Comment:
Can we avoid the reference to parameter `$param`? Return type of method `build` should be typed